Chapters/FirstContact/FirstContactShrunk.md

```diff
@@ -155,13 +153,13 @@ CustomerSatisfactionPresenter >> initializeWindow: aWindowPresenter
 ```
 
 
-You are free to use helper method to return the title and extent of your widget. Now if you reopen your widget, you should get the one displayed in Fig. *@figSecondCut@*.
+You are free to use helper methods to return the title and extent of your widget. Now if you reopen your widget, you should get the one displayed in Fig. *@figSecondCut@*.
 
 
 ![A final version of the customer satisfaction UI.](figures/SecondCut.png width=50&label=figSecondCut)
 
 
-To open a UI, an instance of the class needs to be created and it needs to be sent the `open` message. This will open a window and return an instance of `SpWindowPresenter`, which allows the window to be closed from code.
+Sending the `open` message to a presenter will open a window and return an instance of `SpWindowPresenter`, which allows the window to be closed from code.
```

Comment:
The original text was a repetition of what had been written earlier in the chapter. No need to repeat it.